WinFormDemo/WinFormDemo.csproj

```diff
@@ -151,6 +151,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>copy /Y  "$(SolutionDir)lib\$(Platform)\FTD2XX.dll" "$(TargetDir)"</PostBuildEvent>
+    <PostBuildEvent>copy /Y  "$(SolutionDir)lib\$(Platform)\FTD2XX.dll" "$(TargetDir)"
+copy /Y  "$(SolutionDir)lib\$(Platform)\ATMCD*.dll" "$(TargetDir)"</PostBuildEvent>

```

Comment:
These were also added to Setup32 and Setup64, not that you can easily see that in the .csproj diffs.  Should probably also add to SetupAnyCPU etc (not yet done).